chain/indexer/src/streamer/streamer.rs

```diff
@@ -226,11 +222,16 @@ async fn build_streamer_message(
             .await?,
         );
 
+        // Prepending local receipts with common receipts from current chunk
+        // local ones will be required below to attach them to corresponding execution outcomes
+        let mut chunk_receipts = local_receipts.clone();
```

Comment:
You don't need to accumulate and clone the local receipts. Just prepend the result of `convert_transactions_sir_into_local_receipts` to the non-local receipts (let's rename the field while we unpacking:

```rust
let views::ChunkView { transactions, author, header, receipts: non_local_receipts } = chunk;

let local_receipts = convert_transactions_sir_into_local_receipts(...).await?;
let mut receipts = local_receipts;
receipts.extend(non_local_receipts);
```